hub/config.go

```diff
@@ -84,9 +85,11 @@ func SetFlags(fs *pflag.FlagSet, v *viper.Viper) {
 	fs.BoolP("demo", "D", false, "enable the demo mode")
 	fs.StringP("log-format", "l", "", "the log format (JSON, FLUENTD or TEXT)")
 	fs.BoolP("subscriptions", "s", false, "dispatch updates when subscriptions are created or terminated")
-	fs.BoolP("metrics", "m", false, "enable metrics")
-	fs.StringP("metrics_login", "", "mercure", "the user login allowed to access metrics")
-	fs.StringP("metrics_password", "", "", "the user password allowed to access metrics")
+
+	fs.Bool("metrics-enabled", false, "enable metrics")
+	_ = v.BindPFlag("metrics_enabled", fs.Lookup("metrics-enabled"))
```

Comment:
Why are these calls to `BindPFlag` necessary? It's done automatically line 94.

Reply:
They are not, leftovers from the dot notated version. Done